example/fire-select.js

```diff
@@ -116,7 +116,7 @@ Vue.component('fire-select', {
 
     filters: {
         highlight: function highlight(value) {
-            return this.input.length ? value.replace(new RegExp('(' + this.input + ')', 'g'), '<b>$1</b>') : value;
+            return this.input.length ? value.replace(new RegExp('(' + this.input + ')', 'i'), '<b>$1</b>') : value;
```

Comment:
as far as i remember, without `g` it will only replace the first occurrence, confirm there if this is true... like search for letter `a` and see if all `a` are highlighted.